### ISSUE TITLE
Mysql DDL table definitions use different default charsets: latin1 and utf8

### ISSUE THREAD

**Issue by chrisinmtown** [MISP/MISP#6193]
I believe there's a minor bug lurking in the use of both latin1 and utf8 default character sets in the file MISP/INSTALL/MYSQL.sql. See an excerpt of the file below, showing that table event_tags defaults to latin1 but table favourite_tags defaults to utf-8. As long as everyone just uses ASCII problems will not appear, but problems will occur if people use high-code-page glyphs.

Thanks for considering it.  I will be happy to submit a PR to change latin1 to utf-8 if you agree that this is a bug.

### Work environment

| Questions                 | Answers
|---------------------------|--------------------
| Type of issue             | Bug
| OS version (server)       | RedHat, ...
| OS version (client)       | RHEL 7.4
| PHP version               | 7.2
| MISP version / git hash   | 2.4.XX, 48ddecfd025e1c3ba32086ccef5144c4844d776d
| Browser                   | n/a

### Expected behavior

I think all tables should simply use utf-8. 

### Actual behavior

Table definition varies.

### Steps to reproduce the behavior

View the file MISP/INSTALL/MYSQL.sql.

### Logs, screenshots, configuration dump, ...

```
--
-- Table structure for `event_tags`
--

CREATE TABLE IF NOT EXISTS `event_tags` (
  `id` int(11) NOT NULL AUTO_INCREMENT,
  `event_id` int(11) NOT NULL,
  `tag_id` int(11) NOT NULL,
  `local` tinyint(1) NOT NULL DEFAULT 0,
  PRIMARY KEY (`id`),
  INDEX `event_id` (`event_id`),
  INDEX `tag_id` (`tag_id`)
) ENGINE=InnoDB DEFAULT CHARSET=latin1;

-- -------------------------------------------------------

--
-- Table structure for `favourite_tags`
--

CREATE TABLE IF NOT EXISTS `favourite_tags` (
  `id` int(11) NOT NULL AUTO_INCREMENT,
  `tag_id` int(11) NOT NULL,
  `user_id` int(11) NOT NULL,
  PRIMARY KEY (`id`),
  INDEX `user_id` (`user_id`),
  INDEX `tag_id` (`tag_id`)
) ENGINE=InnoDB DEFAULT CHARSET=utf8;
```

**Comment by mokaddem** [comment]
Thanks for the report!
Fortunately, it's not an issue in the current state as these tables only relies on integers IDs.
We will get rid of both of these charset in favor of a better one for the next major version of MISP

**Comment by chrisinmtown** [comment]
I highly recommend using utf8 for all the tables, and migrate away from latin1.  I don't know of a better choice than utf8.

**Comment by iglocska** [comment]
Indeed, latin1 sounds like a mistake (probably no specific charset defined and exported with the system default). In the future we will move all tables to utf8mb4, but due to the upgrade being pretty resource / time intensive on large instances we're planning on doing all similarly large modifications for the same update.

**Comment by chrisinmtown** [comment]
I guess the change of charset from utf8 and latin1 to a uniform utf8mb4 did not happen yet. Posting here so this does not go stale :/
